tutorials/tmva/TMVA_SOFIE_RDataFrame_JIT.C

```diff
@@ -1,4 +1,4 @@
-/// \file
+std::string inputFile = gROOT->GetTutorialDir() + "/tmva/data/" + i/// \file
```

Comment:
Why this line?? It is probably added by mistake

Reply:
good catch. Will correct - it was added by `sed`.